### PR TITLE
#1210 Set query parameters for /event endpoint as required

### PR DESCRIPTION
**Pull request by johannes-b**
Set the query parameters `keptnContext` and `type` for api endpoint GET `v1/event` since this endpoint is designed to get an event of a particular type from a certain keptnContext.

### CODE REVIEW

File: api/swagger.yaml [hunk 1]
```diff
@@ -52,17 +52,17 @@ paths:
...
       tags:
         - Event
-      summary: Get the latest event matching the parameters
+      summary: Get the latest event matching the required query parameters
       parameters:
         - name: keptnContext
           in: query
           type: string
-          required: false
+          required: true
```
Comment (agrimmer) [inline api/swagger.yaml]: Are you sure that **always** both, the `keptnContext` as well as the `type` are required fields?
Would it be a valid use case to only have the latest event of e.g. a specific `keptnContext`?

File: api/handlers/event.go [hunk 2]
```diff
@@ -68,16 +68,17 @@ func PostEventHandlerFunc(params event.PostEventParams, principal *models.Princi
 // GetEventHandlerFunc returns an event specified by keptnContext and eventType
 func GetEventHandlerFunc(params event.GetEventParams, principal *models.Principal) middleware.Responder {
 
-	logger := keptnutils.NewLogger(*params.KeptnContext, "", "api")
+	keptnContext := uuid.New().String()
```
Comment (agrimmer) [inline api/handlers/event.go]: Do we really need a new `keptnContext` only for the logging? I do not think so.

File: api/swagger.yaml (hunk 1 above)
Comment (johannes-b) [reply api/swagger.yaml]: This is intentional.

File: api/handlers/event.go (hunk 2 above)
Comment (johannes-b) [reply api/handlers/event.go]: Reverted the change to use the provided KeptnContext for logging.

**Pull request merged**